client/src/app.css

```diff
@@ -0,0 +1,18 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+* {
+  border: none;
+  text-decoration: none;
+  border-style: none;
+}
+.App {
+  text-align: center;
+  width: 100%;
+}
+
+body {
```

Comment:
Hey Kristina,

Could you please remove the body color? We will now be controlling the color from a Tailwind folder. This evening, we can choose all the themes and colors with the team.

Also, I will check if the 100% width is affecting any other CSS we have in the repository.

the last one please, could you remove the App class we will try to keep every CSS in it is own component folder